media/js/mozorg/contact-spaces.js

```diff
@@ -408,6 +409,7 @@
             var state = mozMap.getMapState();
             if (state === 'spaces') {
                 initContentId = $('#nav-spaces li.current').data('id');
+                mozMap.showSpace(state.url, null);
```

Comment:
Looks like this causes a redundant ajax request, would it be better just to show the correct marker/map position here instead?